package-scripts.js

```diff
@@ -11,6 +11,7 @@ const {
 const env = require("env-variable")();
 const joinPath = require("path.join");
 const cwd = require("cwd")();
+const config = require('./dist/config.js').config;
```

Comment:
What is dist/config.js going to contain at this point?  There is no application context here.

Reply:
it loads the configuration data into the nconf object to be used in package-scripts.js

Reply:
What if the user wants to change the configuration data, like the current network?  I don't think it is a good idea to require that users alter configuration files within a package installation folder -- the installation may change and then the config is overwritten.

Reply:
they can add a daostack.json file as above, or use command line arguments or environment variables

Reply:
Where would command line arguments be supplied?

Are you saying the user can use environment variables instead of, or in addition to, daostack.json?  Which would take precedence?

Can we be sure this is all documented somewhere?

Just to confirm:  Have you tested using all three methods, both when running in a package script and when running in a straight application (not in nodejs)?

Reply:
Where would command line arguments be supplied?
Are you saying the user can use environment variables instead of, or in addition to, daostack.json? Which would take precedence?
Can we be sure this is all documented somewhere?
Just to confirm: Have you tested using all three methods, both when running in a package script and when running in a straight application (not in nodejs)?

Reply:
command line arguments take the most precendence, then env variables, then the config file. i added more documentation to the README